src/core/shape/2d_primitives.js

```diff
@@ -106,7 +106,8 @@ p5.prototype._normalizeArcAngles = function(
  * origin may be changed with the <a href="#/p5/ellipseMode">ellipseMode()</a> function.<br><br>
  * The arc is always drawn clockwise from wherever start falls to wherever stop falls on the ellipse.
  * Adding or subtracting TWO_PI to either angle does not change where they fall.
- * If both start and stop fall at the same place, a full ellipse will be drawn.
+ * If both start and stop fall at the same place, a full ellipse will be drawn.Be ware that the the
```

Comment:
while removing the extra empty line at the bottom as per @lmccart's suggestion can you also fix the small typo above ie "Be ware" to "Be aware"?Looks good otherwise though thanks!